global.json

```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "8.0.100-rc.1.23463.5",
+    "version": "8.0.100",
```

Comment:
This should already be in main since https://github.com/dotnet/roslyn/pull/70639 automerged.